src/main/java/universe/jdk.astub

```diff
@@ -1,5 +1,16 @@
 import universe.qual.Any;
 
+package java.io;
+
+class PrintStream {
+  public void print(@Any char[] s);
+  public void print(@Any String s);
+  public void print(@Any Object obj);
+  public void println(@Any char[] x);
```

Comment:
```suggestion
  public void print(char @Any [] s);
  public void print(@Any String s);
  public void print(@Any Object obj);
  public void println(char @Any [] x);
```

Reply:
You want the `@Any` on the array itself, not on the component type.

Reply:
Ah, of course. Thanks for the correction.